community/governance.md.tmpl

```diff
@@ -46,6 +46,7 @@ Here are defined the primary teams participating in conda-forge activities.
   emeritus-core. Every 12 months, one core member should ask the rest of
   the team to report their activity status. The question must only include two
   answers: active or inactive. Core members must reply within four weeks.
+  The question should be asked in a public space (e.g. a Github issue/PR).
```

Comment:
Can we specify exactly what we want here? The paragraph says to use the time-out quorum rules below and this is getting confusing.

Reply:
I was trying to clarify @h-vetinari's question here: https://github.com/conda-forge/conda-forge.github.io/pull/2501#pullrequestreview-2779618498

Maybe I should just say that this is a standard (non-sensitive) vote? Or what's the confusion about?

Reply:
The text says

> Individual and group activity inquiries must
  at least follow the same contacting methods stated for Time-out Quorum Rules.

and also

> The question should be asked in a public space (e.g. a Github issue/PR).


Those two things are not exactly the same as the quorum rules require notices to private email lists and chat channels. The quorum rules also specify a bump on the github issue, so I guess it all makes sense. 

However, it is rather confusing.

Reply:
Hm, I guess I should disambiguate "place where the vote is recorded" from "communication channels used to announce and remind about the vote".